lib/plugins/aws/package/lib/mergeIamTemplates.js

```diff
@@ -1,14 +1,13 @@
 'use strict';
 
 const _ = require('lodash');
-const BbPromise = require('bluebird');
 const path = require('path');
 
 module.exports = {
-  mergeIamTemplates() {
+  async mergeIamTemplates() {
     // resolve early if no functions are provided
     if (!this.serverless.service.getAllFunctions().length) {
-      return BbPromise.resolve();
+      return;
```

Comment:
This function should be simply refactored to be a _sync_ function, but all such refactors best if are done separetely in context of other dedicated PR's